en/auth.json

```diff
@@ -140,7 +143,9 @@
       "password_input_error_too_short": "Please provide a password at least 8 characters long",
       "continue_button": "Continue",
       "cant_sign_in_description": "Can't sign in?",
-      "cant_sign_in_link": "Reset password"
+      "cant_sign_in_link": "Reset password",
+      "auth_error": "Error with authentication",
+      "password_confirm_input_label": "Confirm password"
```

Comment:
@viv-kinde there shouldn't be a confirm on this screen - just the setup password